base.js

```diff
@@ -490,7 +490,6 @@ module.exports = {
     // * it's better to use one syntax so the await syntax should be our standard
     'promise/prefer-await-to-callbacks': 'warn', // https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/prefer-await-to-callbacks.md
     // * it's better to use one syntax so the await syntax should be our standard
-    'promise/prefer-await-to-then': 'warn', // https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/prefer-await-to-then.md
```

Comment:
I would remove the comment, too.

Reply:
Also I would leave the rule in the file and just turn it `"off"`. If we change our mind later, we do not have to search for the rule in the docs. What do you mean?